test/check-machines-nics

```diff
@@ -85,6 +82,34 @@ class TestMachinesNICs(VirtualMachinesCase):
         b.wait_in_text("#vm-subVmTest1-network-3-source", "virbr0")
         b.wait_in_text("#vm-subVmTest1-network-3-ipv4-address", "192.168.122.")
 
+        # Check Plug and unplug NICs
+        def checkPlugAndUnplugNICs():
```

Comment:
Can you create a new test for it instead of putting in it in the existing one?

Reply:
Sure, re-push it